recipes/faunus/conda_build_config.yaml

```diff
@@ -0,0 +1,23 @@
+python:
+    - 3.6
+    - 3.7 
+mpi:
+    - mpich
+    - openmpi
+    - nompi
+pin_run_as_build:
+    mpich: 3.2.*
```

Comment:
pin_run_as_build is only for masks such as `x.x` or `max_pin: x.x; min_pin: x.x.x` to create a `run` dependency pinned based on the version installed in `build` or `host`. To pin a specific version of a package, use the top-level key, as is being done here for `python`:

```yaml
mpi:
  - mpich
  - openmpi
  - nompi
mpich: 3.2.*
openmpi: 3.0.*
```

Reply:
OK, pin_run_as_build removed.